semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/NewTagNodePairTraversal.scala

```diff
@@ -3,9 +3,11 @@ package io.shiftleft.semanticcpg.language
 import io.shiftleft.codepropertygraph.generated.EdgeTypes
 import io.shiftleft.codepropertygraph.generated.nodes.{NewNode, NewTagNodePair, StoredNode}
 import io.shiftleft.passes.DiffGraph
+import overflowdb.BatchedUpdate.DiffGraphBuilder
 import overflowdb.traversal._
 
 class NewTagNodePairTraversal(traversal: Traversal[NewTagNodePair]) extends HasStoreMethod {
+  val original = traversal
```

Comment:
Is this used somewhere?

Reply:
yesss, here https://github.com/joernio/joern/pull/1801/files#diff-3991921d186ab499e2c9c975dea554375dc7ec17624aed0cd22094e0cc7ab968R24

Reply:
If the idea here is to be able to reaccess the traversal after it has been used, then you need a clone here.

Reply:
that val was only temporary, forgot to add a comment about it. removing it soon with https://github.com/joernio/joern/pull/1814/files